src/lib/progress-spinner/progress-spinner.md

```diff
@@ -21,3 +21,7 @@ In "indeterminate" mode, the `value` property is ignored.
 ### Theming
 The color of a progress-spinner can be changed by using the `color` property. By default, 
 progress-spinners use the theme's primary color. This can be changed to `'accent'` or `'warn'`.
+
+### Accessibility
+Each progress spinner should be marked with an `aria-label` or `aria-labelledby` with a meaningful
```

Comment:
I think here and in progress bar, you mean "marked by aria-label or aria-labelledby with a meaningful label".